src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (salary) => {
+  return salary.replace(/\D/g, '');
+};
+
+const sortList = (list) => {
+  list
+    .sort(
+      (value1, value2) =>
+        dataAtributToNumber(value2.dataset.salary) -
+        dataAtributToNumber(value1.dataset.salary),
+    )
+    .forEach((employee, i) => list[i].parentElement.appendChild(employee));
```

Comment:
In this line, you are reordering the elements in the DOM. This operation is expensive in terms of performance. It's better to sort the data first and then render it.